### PR TITLE
Confirmation Page content

### LINKED ISSUE #679: Campaign: Confirmation template
Create a page callback for confirmation of reportback submission.

Update Campaign's reportback submission to redirect user to this page upon success.

### PR DESCRIPTION
**Pull request by aaronschachter**
Closes #679
- Moves the dosomething_campaign module's reportback-confirmation.tpl.php file into a /theme subdirectory
- Creates an override reportback-confirmation.tpl.php in the dosomething_paraneue/templates/campaign directory
- Adds a bunch of `dosomething_campaign_get_recommended` functions to return campaigns which the user has not signed up for.
  - First finds any staff picks that the user has not signed up for
  - If less than 3, finds any campaigns which share the same Primary Cause that user has not signed up for
- Renders them as recommended campaigns on the Campaign Reportback Confirmation page.
- Adds documentation

### CODE REVIEW

File: lib/modules/dosomething/dosomething_campaign/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+The confirmation page displays 3 recommended campaigns for the user.  This is
+determined by first finding published campaigns which are staff picks, that the
+user has not signed up for.
+
+If there are less than 3 staff picks that the user hasn't signed up for, the
+confirmation page will also display published campaigns which share the same
+Primary Cause, for which the user has not signed up for.
+
+
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_campaign/README.md:61]: :+1:

File: lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module [hunk 2]
```diff
@@ -96,32 +96,73 @@ function dosomething_campaign_reportback_confirmation_page_access($node) {
...
 /**
+ * Returns an array of variables for a given recommended campaign $nid.
+ *
+ * @param int $nid
+ *   The node $nid to return variables for.
+ *
+ * @return array
+ *   An array of variables.
+ */
+function dosomething_campaign_get_recommended_campaign_vars($nid) {
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module:136]: Small thing, but perhaps the function should be something more generic like 'campaign_teaser_vars'
Actually, it's pretty similar to the pitch page variables.

Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module:136]: The pitch page would be a different crop I'd think... which would make sense to keep in a separate "recommended" function.  I'm happy to rename later if we want to reuse it, what do you think?

Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module:136]: Discussed with @angaither -- we can visit building a generic teaser vars function in a later PR.

**Pull request merged**